service-base/src/main/java/org/eclipse/hono/service/plan/PrometheusBasedResourceLimitChecksConfig.java

```diff
@@ -0,0 +1,178 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package org.eclipse.hono.service.plan;
+
+import java.util.Objects;
+
+import org.eclipse.hono.util.PortConfigurationHelper;
+
+/**
+ * The configuration properties required for the PrometheusBasedResourceLimitChecks.
+ */
+public class PrometheusBasedResourceLimitChecksConfig {
```

Comment:
if this class is not supposed to be extended, make it final, please. Otherwise, please make the methods final ...

Reply:
Now it is final.